data/release/dbus/com.redhat.lightspeed.conf

```diff
@@ -2,25 +2,34 @@
 <!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-Bus Bus Configuration 1.0//EN"
  "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
 <busconfig>
-  <!-- Allow the command line assistant service to own the name -->
+  <!-- Deny everyone by default -->
```

Comment:
The deny by default is unnecessary. The system D-Bus has a deny-all by default policy. It doesn't hurt to keep this here, but it's not strictly required.

Reply:
removed the deny rules